data/api/client-server/relations.yaml

```diff
@@ -84,10 +332,9 @@ paths:
             The pagination token to start returning results from. If not supplied, results
             start at the earliest topological event known to the server.
 
-            Can be a `next_batch` or `prev_batch` token from a previous call, or an equivalent
-            token from [`/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages)
-            or [`/sync`](/client-server-api/#get_matrixclientv3sync) to limit results to the
-            events returned by that section of timeline.
+            Can be a `next_batch` or `prev_batch` token from a previous call, or a returned
```

Comment:
I don't understand why it is useful to pass a `prev_batch` from a previous call here (at until MSC3716 lands).

Likewise above x2

Reply:
fair, though the fact we explicitly permit it does rather imply that it might be a useful thing to do, which it's not.

Reply:
removed the end batch tokens manually